brave-intl/publishers
# [Prod] improvement: Remove wordpress verification logic

## Pull Request Name

Remove wordpress domain identification

#### Features

Removes auto detection for wordpress.com domains and treats them like any other website.

#### How To Test

1. Add a new channel as a website
2. Add in wordpress.com as the domain
3. In staging note that it requires a DNS verification step while in prod it takes you through a wordpress pathway.

#### Pull Request Checklist
